lib/shoryuken/queue.rb

```diff
@@ -14,8 +14,8 @@ def initialize(client, name)
 
     def visibility_timeout
       client.get_queue_attributes(
-        queue_url: url,
-        attribute_names: ['VisibilityTimeout']
+          queue_url:       url,
+          attribute_names: ['VisibilityTimeout']
       ).attributes['VisibilityTimeout'].to_i
```

Comment:
So then we'll make the current style the code style. I think it's beneficial to include so new contributors can write it correctly going forward, and it saves you time :)